corehq/apps/reports/views.py

```diff
@@ -957,6 +957,19 @@ def rebuild_case_view(request, domain, case_id):
 @require_case_view_permission
 @require_permission(Permissions.edit_data)
 @require_POST
+def resave_case(request, domain, case_id):
+    case = get_document_or_404(CommCareCase, domain, case_id)
+    CommCareCase.get_db().save_doc(case._doc)  # don't just call save to avoid signals
```

Comment:
@gcapalbo I'm intentionally not calling `.save()` so that the reminder logic doesn't refire. if you think it would actually be good for those to get called we can easily switch it.